test/games/strategy/net/MessengerTest.java

```
@@ -325,8 +326,8 @@ public INode getLastSender() {
   private void waitForMessage() {
     try {
       lock.wait(1500);
-    } catch (final InterruptedException ie) {
-      ie.printStackTrace();
+    } catch (final InterruptedException e) {
+      ClientLogger.logQuietly(e);
```

Actually, we should fail the test since it is an unexpected exception.